docs/docs/external/getting-started/test-network/setup-packages.md

```diff
@@ -0,0 +1,346 @@
+
+Create a personal access token with `read:packages` access in github in order to use modules published in github packages. Refer [Creating a Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) for help.
+
+## Getting the Code and Documentation
+
+Clone the [weaver-dlt-interoperability](https://github.com/hyperledger-labs/weaver-dlt-interoperability) repository. The code to get a basic test network up and running and test data-sharing interoperation flows lies in the subfolder `tests/network-setups`, which should be your starting point, though the setups will rely on other parts of the repository, as you will find out in the instructions given on this page.
+
+## Hyperledger Fabric Components
+
+Using the sequence of instructions below, you can start two separate Fabric networks, each with a single channel and application contract (chaincode). You can also start an interoperation contract, a relay and a _driver_ acting on behalf of each network. You can build a Fabric CLI tool with which you can initialize both networks' ledgers with access control policies, foreign networks' security groups (i.e., membership providers' certificate chains), and some sample key-value pairs that can be shared during subsequent interoperation flows.
+
+### Fabric Network
+
+The code for this lies in the `tests/network-setups` folder.
+
+This folder contains code to create and launch networks `network1` and `network2` of identical specifications:
+- Network: 1 peer, 1 peer CA, 1 ordering service node, 1 ordering service CA
+- Single channel named `mychannel`
+- Single contract named `simplestate` (deployed on `mychannel`) that supports simple transactions (`Create`, `Read`, `Update`, `Delete`) involving storage and lookup of <key, value> pairs.
+
+Follow the instructions below to build and launch the networks:
+- Navigate to the `tests/network-setups/fabric/dev` folder.
+- To spin up both network1 and network2 with interoperation chaincode installed, run:
+  ```
+  make start-interop
+  ```
+- (_Note_: If you do not wish to test Fabric-Fabric interoperation, you can choose to install only one of the two networks along with its interoperation chaincode. For `network1`, run `make start-interop-network1`, and for `network2`, run `make start-interop-network2`.)
+
+For more information, refer to the associated [README](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/tests/network-setups/fabric/dev).
+
+**Troubleshooting Tips**:
+- If you see any errors during the launches, re-check the prerequisites (software installations and credentials). Ensure your network connection is working. As a safe bet, you can retry after cleanup: kill and remove all Docker containers and associated volumes.
+- If `protoc` or `protoc-gen-go` throws an error, reinstall `protoc` and `protoc-gen-go` using suggestions made in the Prerequisites section above.
```

Comment:
This step can also be removed, as `protoc` isn't used when using github packages.

Reply:
Fixed